arch/x86/Makefile.in

```diff
@@ -11,7 +11,8 @@ SUFFIX=
 AVX2FLAG=-mavx2
 SSE2FLAG=-msse2
 SSSE3FLAG=-mssse3
-SSE4FLAG=-msse4
+SSE41FLAG=-msse4.1
+SSE42FLAG=-msse4.2 -mcrc32
```

Comment:
Clang doesn't like `-mcrc32`.

Reply:
Hmm I stumbled onto gcc failing to build this due to use native being flipped in. Still, likely gcc won't struggle with this with -msse4.2, even though it's less specific. I'll change it back.